README.md

```diff
@@ -1,6 +1,133 @@
-# website
+# Website
+
 The website for the Civic Tech Index
 
+To develop the site, you'll need to first clone the repository on to your computer. For new Git users, see the [Using Git](#using-git) section below. <br><br>
+
+# OVERVIEW
+**Set up**
+1. [Join the Repo Team](#step-1-become-a-member-of-the-repository-team)
+
+2. [Using Git](#using-git) and [Fork the Repo](#step-2-fork-the-repository)
+
+3. [Clone to your local machine](#step-3-clone-your-online-repository-to-your-local-computer)
+
+4. [Set up Docker](#step-4-setting-up-docker)
+
+**Before you start working on an issue**
+
+5. [Read CTI's Site Architecture to get acquainted with how the website is structured](https://github.com/CTI-website-frontend/website/wiki/CTI-Site-Architecture)
+
+6. [Switch to new issue branch before you start making changes](#step-6-change-to-a-new-branch)
+
+
+**After you've worked on your issue and before you make a pull request:**
+
+7. [Check upstream before you push](#step-7-check-upstream-before-you-push).
+
+8. [No changes in the upstream repo](#step-7a-no-changes-in-the-upstream-repository)
```

Comment:
This is item number 8 but links to Step 7a. Should they match?

Reply:
Fixed this issue